### PR TITLE
Add setting to substitute substrings in description_head for item attributes

### PR DESCRIPTION
**Pull request by JasperCraeghs**

### CODE REVIEW

**Comment by Ben2022** [comment]
Thanks!

**Comment by Ben2022** [comment]
I cannot merge, can you reassign please?

File: README.rst [hunk 1]
```diff
@@ -41,7 +41,8 @@ Configuration
         'default_project': 'SWCC',
         'relationship_to_parent': ('depends_on', r'MEETING-[\w_]+'),
         'components': '[SW],[HW]',
-        'description_head': 'Action raised in meeting.\n\n',
+        'description_head': 'Action raised in [this meeting|https://docserver.com/<<file_name>>.html].\n\n',
+        'str_to_attr_mapping': {'<<file_name>>': 'docname'},
```
Comment (Letme) [inline README.rst:45]: `description_str_to_attr` might be better.

**Comment by Letme** [comment]
@Ben2022  you can go to diff->review->approve :wink:

**Review by Ben2022 — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (JasperCraeghs) [reply README.rst:45]: Definitely! Also, the main description for the Jira ticket can benefit from this feature (not only description_head).

Comment (JasperCraeghs) [reply README.rst]: Done in ab29d44b28f4b66f702b6f1e34168916480356b6

**Pull request merged**